bootstrap.py

```diff
@@ -1,5 +1,125 @@
 import argparse
+import errno
+import os
+import os.path
 
+api_template = """
+module {module_name}.API where
+
+import Html exposing (Html)
+import StartApp
+import Effects exposing (Never)
+import Task exposing (Task)
+
+import {module_name}.Update exposing (update, Action(..), Addresses)
+import {module_name}.Model exposing (Model)
+import {module_name}.View exposing (view)
+
+
+app : StartApp.App Model
+app =
+    let
+        initModel : Model
+        initModel =
+            {{ }}
+
+        modelWithEffects =
+            (initModel, Effects.none)
+
+        addresses =
+            {{ }}
+    in
+        StartApp.start
+            {{ init = modelWithEffects
+            , view = view
+            , update = (update addresses)
+            , inputs = [ ]
+            }}
+
+main : Signal Html
+main =
+    app.html
+
+port authToken : String
+
+port tasks : Signal (Task.Task Never ())
+port tasks =
+    app.tasks
+""".lstrip()
+
+model_template = """
+module {module_name}.Model where
+
+type alias Model =
+  {{ }}
+""".lstrip()
+
+update_template = """
+module {module_name}.Update where
+
+import Effects exposing (Effects)
+import {module_name}.Model exposing (..)
+
+type Action
+  = NoOp
+
+type alias Addresses =
+  {{ }}
+
+update : Addresses -> Action -> Model -> (Model, Effects.Effects Action)
+update addresses action model =
+  case action of
+    NoOp ->
+      (model, Effects.none)
+""".lstrip()
+
+view_template = """
+module {module_name}.View (..) where
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (..)
+import Signal
+
+import {module_name}.Model exposing (..)
+import {module_name}.Update exposing (..)
+
+
+view : Signal.Address Action -> Model -> Html
+view address model =
+  div [] [ text "Hello world!" ]
+""".lstrip()
+
+templates = {'API.js.elm': api_template,
+             'Model.elm': model_template,
+             'Update.elm': update_template,
+             'View.elm': view_template}
+
+def make_boilerplate(module_name, directory, template, file_name):
+    file_path = os.path.join(directory, file_name)
+    templated = template.format(module_name=module_name)
+
+    with open(file_path, 'w') as f:
+        f.write(templated)
+
+def make_boilerplates(module_name, directory, templates):
+    for file_name, template in templates.items():
```

Comment:
If I remember right, this makes a list instead of an iterator. Is that going to be an issue? There should only be a few items here anyway.

It's compatible with py2 and py3 this way though.